extensions/wikia/PortableInfobox/PortableInfobox.i18n.php

```diff
@@ -5,11 +5,31 @@
 	'portable-infobox-desc' => 'Create portable infoboxes which can be rendered using clean semantic HTML markup on
 	any skin / platform using using easy to understand powerful XML-like markup',
 	'unimplemented-infobox-tag' => 'Unimplemented infobox tag: <$1>',
-	'xml-parse-error' => 'Incorrect XML markup. Please validate your XML.'
+	'xml-parse-error-info' => 'There is a problem with parsing the infobox:',
+	'xml-parse-error' => 'Incorrect XML markup. Please validate your XML',
+	'xml_err_document_end' => 'Extra content at the end of the document',
+	'xml_err_undeclared_entity' => 'Undeclared entity (for example: &nbsp;)',
+	'xml_err_attribute_not_started' => 'Attribute value should be inside quotation marks',
+	'xml_err_attribute_without_value' => 'Tag attribute needs to have value',
+	'xml_err_space_required' => 'Attributes construct error',
+	'xml_err_name_required' => 'Error parsing attribute name',
+	'xml_err_gt_required' => "Tag definition is not properly terminated",
+	'xml_err_tag_name_mismatch' => 'Opening and ending tag mismatch',
+	'xml_err_tag_not_finished' => 'Premature end of tag'
 ];
 
 $messages[ 'qqq' ] = [
 	'portable-infobox-desc' => 'Portable Infobox extension description',
 	'unimplemented-infobox-tag' => 'Error message for using unimplemented infobox tag; $1 is the tag name without pointy braces',
-	'xml-parse-error' => 'Error message for providing incorrect XML markup'
+	'xml-parse-error-info' => 'General message for parsing problem',
```

Comment:
Just caught this - please make sure that we have prefixes for all messages; portable-infobox-*